glossary.yml

```
@@ -6060,8 +6060,8 @@
   ja:
     term: "変数 (プログラム)"
     def: >
-      プログラム内にデータが結合するの名前です。変数の値は定義したの後で代再入出来ます。
-      変数は英語で「variable」(ヴァリアブル)と呼んでいます。
+      プログラム内にデータが結合するの名前です。変数の値は定義したの後で代再入出来のこと。
+      変数は英語で「variable」(ヴァリアブル)と呼んでいのこと。
 
```

I understand this sentence is specifically added for Japanese. If it should stay, I would suggest to modify:
変数は英語で「variable」(ヴァリアブル)という / 呼ぶ (whichever you prefer)。

---

Not sure about the scope of glosario but I've seen a lot of Japanese definitions give the English term, especially for technical jargon as Japanese users will likely encounter them in English documentation.

Change to plain form as suggested.

```suggestion
      変数は英語で「variable」(ヴァリアブル)という。
```

---

I saw a conversation with @naoe-tatara at https://github.com/carpentries/glosario/pull/303#discussion_r614480720. I feel it may be easier to avoid mentioning a pronunciation in other languages for now. When Japanese is interested to know how to say a word in English, they should check with glosario en entry or Google. Let's keep each entry simple.